Tests/Integration/Network/Connection/FogSyncErrorConnectionIntTests.swift

```diff
@@ -48,156 +51,162 @@ class FogSyncErrorConnectionIntTests: XCTestCase {
             consensusBlockIndex: consensusIndex,
             transportProtocol: transportProtocol,
             expectation: expect,
-            shouldSucceed: true
+            shouldSucceed: shouldSucceed
         )
     }
     
-    func testFogBehindConsensusWithinThreshold() throws {
+    func testFogBehindConsensusWithinDelta() throws {
         try testSupportedProtocols(description: description) {
-            try fogBehindConsensusWithinThresholdAheadConsensus(
+            try fogBehindConsensusWithinDeltaAheadConsensus(
                     transportProtocol: $0,
                     expectation: $1)
         }
     }
     
-    func fogBehindConsensusWithinThresholdAheadConsensus(
+    func fogBehindConsensusWithinDeltaAheadConsensus(
```

Comment:
```suggestion
    func fogBehindConsensusWithinDelta(
```

Reply:
oh ya I see now this was a typo on my part

Reply:
both should be fixed now